src/libraries/System.Private.CoreLib/src/System/SearchValues/Strings/AsciiStringSearchValuesTeddyBucketizedN2.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace System.Buffers
+{
+    internal sealed class AsciiStringSearchValuesTeddyBucketizedN2<TStartCaseSensitivity, TCaseSensitivity> : AsciiStringSearchValuesTeddyBase<SearchValues.TrueConst, TStartCaseSensitivity, TCaseSensitivity>
+        where TStartCaseSensitivity : struct, StringSearchValuesHelper.ICaseSensitivity
+        where TCaseSensitivity : struct, StringSearchValuesHelper.ICaseSensitivity
+    {
+        public AsciiStringSearchValuesTeddyBucketizedN2(string[][] buckets, ReadOnlySpan<string> values, HashSet<string> uniqueValues)
+            : base(buckets, values, uniqueValues, n: 2)
+        { }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I'd hope this wouldn't be necessary.  Is it for some reason?  If we need to be putting AggressiveInlining on things that simply delegate to another method with the exact same argument, our inlining solution is in trouble.

Reply:
I removed a couple of these in f1aca19ecdd15cbfe252018764023cb9b901ad20, inlining for the couple of places I checked looked okay afterward.